content/overview/overview/building-process.md

```diff
@@ -11,15 +11,16 @@ aliases:
 
 ## How long it took to build
 
-After exploring configuration-management based approaches and tools, we turned instead to the Platform as a Service (PaaS) model. By building on Cloud Foundry, we were able to deploy a functioning PaaS quickly, then fine-tune it with each project we launched on it. The first prototype of the platform took about three months to complete, and production apps started launching about a month after that.
+After exploring configuration-management based approaches and tools, engineers at 18F turned instead to the platform-as-a-service (PaaS) model. By building on Cloud Foundry, the cloud.gov team was able to deploy a functioning PaaS quickly, then fine-tune it with each project an 18F product team launched on it. The first prototype of the platform took about three months to complete, and teams began to deploy their work to production about a month after that.
+launching about a month after that.
```

Comment:
Thank you. I screwed up a merge conflict at one point and accidentally duplicated the conflicts. Thought I'd caught them all.